test/data/test_builtin_datasets.py

```diff
@@ -122,3 +122,61 @@ def test_imdb(self):
         old_vocab = train_dataset.get_vocab()
         new_vocab = Vocab(counter=old_vocab.freqs, max_size=2500)
         new_train_data, new_test_data = IMDB(vocab=new_vocab)
+
+    def test_sequence_tagging(self):
```

Comment:
If the CI tests for datasets don't take a lot of time to run, we should include tests to cover both datasets: `UDPOS` and `CoNLL2000Chunking`.

Reply:
done